src/api/søknadApi.ts

```diff
@@ -28,12 +28,26 @@ export interface OpprettetSøknad {
     søknad: Søknad;
 }
 
-export async function sendSøknad(søknad: SøknadInnhold): Promise<ApiClientResult<OpprettetSøknad>> {
-    return apiClient({ url: '/soknad', method: 'POST', body: søknad });
+export async function sendUføresøknad(søknad: SøknadInnhold): Promise<ApiClientResult<OpprettetSøknad>> {
+    return apiClient({
+        url: '/soknad/ufore',
+        method: 'POST',
+        body: {
+            type: 'ufore',
+            ...søknad,
+        },
+    });
```

Comment:
Url er 'soknad/ufore', men i bodyen spesifiserer vi type. Hva er grunnen til det? Burde ikke routen bare håndtere at det er en uføre søknad?

Reply:
Grunnen er at backend trenger feltet for å kunne deserialisere på et sealed interface. Vi hadde ikke trengt å spesifisere /ufore i url-en sånn _egentlig_, men tror det er smart for å sikre at vi ikke en gang i fremtiden klarer å sende en alderssøknad som uføresøknad og motsatt